### PR TITLE
Use vaadin-date-picker-light directly

### PR DESCRIPTION
**Pull request by awikkerink**

### CODE REVIEW

**Comment by awikkerink** [comment]
FYI @alogan519

File: d2l-datetime-picker.html [hunk 1]
```diff
@@ -57,7 +89,11 @@
 
 		</style>
 		<label for="input">{{_dateLabel}}</label>
-		<d2l-date-picker value="[[_dateValue]]"></d2l-date-picker>
+		<vaadin-date-picker-light value="{{date}}">
+			<iron-input>
```
Comment (mdgbayly) [inline d2l-datetime-picker.html]: Is there a reason to use iron-input rather than just a regular html input?

Comment (awikkerink) [inline d2l-datetime-picker.html]: vaadin-date-picker expects an iron-input. Probably so it can setup binding

Comment (awikkerink) [inline d2l-datetime-picker.html]: https://github.com/vaadin/vaadin-date-picker/blob/master/vaadin-date-picker-light.html#L110

**Comment by mdgbayly** [comment]
@awikkerink I'm going to bump the version to 0.1.0 to make it easier to pick up minor version bumps from other components.

**Pull request merged**